hack/ci-test-e2e.sh

```diff
@@ -12,9 +13,17 @@ generate_junit() {
 }
 trap generate_junit EXIT
 
+if [ -z "${1}" ]
```

Comment:
Please set a default instead so we can keep the `set -u`

```suggestion
if [ -z "${1:-}" ]
```